scripts/process_schemas.py

```diff
@@ -271,7 +271,7 @@ class SchemaProcessor(object):
     """
     def __init__(self, args):
         self.version = args.version
-        self.destinationFile = args.outputFile
+        self.destinationFile = args.output_file
```

Comment:
I know underscore is PEP8 / Pythonic, but because the schema is camel-cased this project is also camel-cased.